morf-core/src/test/java/org/alfasoftware/morf/sql/TestObjectTreeTraverserWithSqlElementVisitor.java

```diff
@@ -103,7 +102,6 @@ public void testSimpleJoinCriterionWithSubquery() {
     verify(callback).visit(select2);
     verify(callback).visit(selectsAsField);
     verify(callback).visit(select2.getFields().get(0));
-    verifyNoMoreInteractions(callback);

```

Comment:
Is it not important to check?

Reply:
I think it makes more calls, it wasn't verified properly with the old Mockito

Reply:
so this will fail with new inline mock maker, it is usually more strict

Reply:
@jsimlo Ok, found a better way, see last commit